servlet/src/main/scala/org/http4s/servlet/Http4sServlet.scala

```diff
@@ -133,7 +133,7 @@ class Http4sServlet(service: HttpService,
   private def toRequest(req: HttpServletRequest): ParseResult[Request] =
     for {
       method <- Method.fromString(req.getMethod)
-      uri <- Uri.requestTarget(s"${req.getRequestURI}?${req.getQueryString}")
+      uri <- Uri.requestTarget(Option(req.getQueryString).map { q => s"${req.getRequestURI}?$q" }.getOrElse(req.getRequestURI))
```

Comment:
[Benchmarks](https://gist.github.com/rossabaker/0bcfd7e855ac542ecc592b32f9460de1).  I always thought map/getOrElse was slow.  Not so, at least here.